doc/rst/technotes/gpu.rst

```diff
@@ -112,89 +107,46 @@ Examples with multiple GPUs:
-this by changing  ``CHPL_GPU_ARCH`` or by passing it to ``chpl`` via
-``--gpu-arch``.
-
-We also suggest setting ``CHPL_RT_NUM_THREADS_PER_LOCALE=1`` (this is necessary
-if using CUDA 10).
-
-To compile a program simply execute ``chpl`` as normal.  To ensure that a loop
-is executing on the GPU you can use the operations in the :mod:`GpuDiagnostics`
-module or use the :proc:`~GPU.assertOnGpu()` proc from the  :mod:`GPU` module.
-
-Requirements and Limitations
-----------------------------
-
-Because of the early nature of the GPU support project there are a number of
-limitations. We provide a (non exhaustive) list of these limitations in this
-section; many of them will be addressed in upcoming editions.
-
-* We currently support NVIDIA and AMD GPUs
+Requirements
+~~~~~~~~~~~~
 
 * ``LLVM`` must be used as Chapel's backend compiler (i.e.
   ``CHPL_LLVM`` must be set to ``system`` or ``bundled``). For more information
   about these settings see :ref:`Optional Settings <readme-chplenv>`.
 
-* If using a ``system`` LLVM it must have been built with support for the
-  relevant target of GPU you wish to generate code for (e.g.  NVPTX to target
-  NVIDIA GPUs and AMDGPU to target AMD GPUs).
+  * If using a ``system`` LLVM it must have been built with support for the
+    relevant target of GPU you wish to generate code for (e.g.  NVPTX to target
+    NVIDIA GPUs and AMDGPU to target AMD GPUs).
 
-* If using a system install of ``LLVM`` we expect this to be the same
-  version as the bundled version (currently 14). Older versions may
-  work; however, we only make efforts to test GPU support with this version.
+  * If using a system install of ``LLVM`` we expect this to be the same
+    version as the bundled version (currently 14). Older versions may
+    work; however, we only make efforts to test GPU support with this version.
 
 * ``CHPL_TASKS=qthreads`` is required for GPU support.
 
-* PGAS style communication is not available within GPU kernels; that is:
-  reading from or writing to a variable that is stored on a different locale
-  from inside a GPU eligible loop (when executing on a GPU) is not supported.
-
-* There is no user-level feature to specify GPU block size on a
-  per-kernel basis. This can be set on a program wide basis at compile-time by
-  passing ``--gpu-block-size=size`` to the compiler or setting it with the
-  ``CHPL_GPU_BLOCK_SIZE`` environment variable.
-
-* The use of most ``extern`` functions within a GPU eligible loop is not supported
-  (a limited set of functions used by Chapel's runtime library are supported). 
-
-   * Various functions within Chapel's standard modules call unsupported
-     ``extern`` functions and thus are not supported in GPU eligible loops.
-
-* Runtime checks such as bounds checks and nil-dereference checks are
-  automatically disabled for CHPL_LOCALE_MODEL=gpu.
-
-* For AMD GPUs:
+* ``nvcc`` (for NVIDIA) or ``hipcc`` (for AMD) must be available.
```

Comment:
How about I revert the order in that sentence to:

```
nvcc or hipcc must be available. Paths of these executables
will be used to find the path for CUDA Driver API or HIP API.
These paths can be overriden by `CHPL_CUDA_PATH` or `CHPL_ROCM_PATH`
```

Reply:
I think someone might still walk away from that sentence thinking we're using the `nvcc` or `hipcc` compiler "under-the-hood". Maybe that's not that big of a deal but it might lead folks to guessing they could get cuda/hip code by using `--savec`.

And (even though it might seem obvious) I think we should also make it clear that the user just needs one of `nvcc` or `hipcc` and not both and more specifically that `nvcc` is only needed for NVIDIA support and `hipcc` for AMD.

How about this?

```
Either nvcc (for NVIDIA support) or hipcc (for AMD support) must be available; Chapel uses libraries included in these packages and will automatically deduce the path to these libraries based on the location of the `nvcc`/`hipcc` executable. Note that the automatically deduced paths may be overwritten by manually setting the `CHPL_CUDA_PATH` or `CHPL_ROCM_PATH` environment variables.
```